### PR TITLE
KV-2021 proposal: HCO - What is New, What to Expect

### PR DESCRIPTION
**Pull request by nunnatsa**
Signed-off-by: Nahshon Unna-Tsameret <nunnatsa@redhat.com>

### CODE REVIEW

File: events/2021-kubevirt-summit/proposals/hco-proposal-2021.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+deploying and controlling KubeVirt and several adjacent operators:
+* Containerized Data Importer
+* Scheduling, Scale and Performance
+* Cluster Network Addons
+* Node Maintenance
+
+The purpose of HCO is to ease the deployment, upgrade, monitoring and configuration of an opinionated version of the he KubeVirt cluster.
+
+In this session will cover the following topics:
+1. Why we think that an opinionated deployment and a single source of truth for the whole product is really important
```
Comment (codificat) [inline events/2021-kubevirt-summit/proposals/hco-proposal-2021.md:14]: Would this point belong more in #57 which covers HCO for the user?

Well, certainly a short overview of the background is welcome, I'm just wondering about potential [too much] overlap here

**Review by codificat — COMMENTED** [review]
Thanks for submitting this proposal!

Another proposal was submitted about HCO for the user track, #57 .  I understand that both proposals target a different track and therefore have different goals and expectations for the audience, but right now I believe that this distinction might not be clear enough. What do you think?

**Comment by codificat** [comment]
Thank you for submitting this session proposal for the KubeVirt summit!

We are working to prepare a schedule that can accommodate as many of the received proposals as possible, according to the capacity available in the event.

We might have to request some adjustments to some of the proposals in order to achieve that goal.

More feedback will be provided next week.

Looking forward to meeting you at the event!

**Comment by davidvossel** [comment]
There ended up being way more content that we expected. As a result, we had to quickly form a review team (thanks Red Hat and NVIDIA) to make some hard decisions about what presentations to accept.

This presentation did not make the acceptance list. Thank you for your submission and support of the KubeVirt community!

**Pull request closed without merging**